source/docs/software/getting-started/3rd-party-libraries.rst

```diff
@@ -1,150 +1,74 @@
- IDE, you will need to manually create a “vendordeps” folder in your\
- project and copy any desired vendor JSON files from the “frcYYYY” folder\
- (where they should be placed by an offline installer) or download them\
- directly from the vendor and place them into the folder in the
- project.**
-
-Adding an offline-installed Library
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-|image0|
-
-To add a vendor library that has been installed by an offline installer,
-press **Ctrl+Shift+P** and type WPILib or click on the WPILib icon in
-the top right to open the WPILib Command Palette and begin typing
-**Manage Vendor Libraries**, then select it from the menu. Select the
-option to **Install new libraries (offline)**.
-
-|image1|
-
-Select the desired libraries to add to the project by checking the box
-next to each, then click OK. The JSON file will be copied to the
-**vendordeps** folder in the project, adding the library as a dependency
-to the project.
-
-Checking for Updates (offline)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Remember: Dependencies are now version managed and done on a per-project
-bases. Even if you have installed an updated library using an offline
-installer, you will need to Manage Vendor Libraries and select **Check
-for updates (offline)** for each project you wish to update.
-
-Checking for Updates (online)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-Part of the JSON file that vendors may optionally populate is an online
-update location. If a library has an appropriate location specified,
-running **Check for updates (online)** will check if a newer version of
-the library is available from the remote location.
-
-Removing a library dependency
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To remove a library dependency from a project, select **Manage Current
-Libraries** from the **Manage Vendor Libraries** menu, check the box for
-any libraries to uninstall and click OK. These libraries will be removed
-as dependencies from the project.
-
-.. |image0| image:: images/3rd-party-libraries/adding-offline-library.png
-.. |image1| image:: images/3rd-party-libraries/library-installer-steptwo.png
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+For C++ and Java a JSON file describing the vendor library is installed on your system to ``~home\frcYYYY\vendordeps`` (~home = ``C:\Users\Public`` on Windows). This can either be done by an offline installer or the file can be fetched from an online location using the menu item in VSCode. This file is then used from VS Code to add to the library to each individual project. Vendor library information is managed on a per-project basis to make sure that a project is always pointing to a consistent version of a given vendor library. The libraries themselves are placed in the Maven cache at ``C:\Users\Public\frcYYYY\maven``. Vendors can place a local copy here with an offline installer (recommended) or require users to be online for an initial build to fetch the library from a remote Maven location.
```

Comment:
```suggestion
For C++ and Java a JSON file describing the vendor library is installed on your system to ``~/frcYYYY/vendordeps`` (~home = ``C:\Users\Public`` on Windows). This can either be done by an offline installer or the file can be fetched from an online location using the menu item in VSCode. This file is then used from VS Code to add to the library to each individual project. Vendor library information is managed on a per-project basis to make sure that a project is always pointing to a consistent version of a given vendor library. The libraries themselves are placed in the Maven cache at ``C:\Users\Public\frcYYYY\maven``. Vendors can place a local copy here with an offline installer (recommended) or require users to be online for an initial build to fetch the library from a remote Maven location.
```